### PR TITLE
feat: organized the files and resources

### PR DESCRIPTION
**Pull request by vinzvinci**

### CODE REVIEW

File: learning-guide/beginner.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+About Blockchain, web 3 and somewhat related to Dapp.
+
+| Title | Content | 
+| :-- | --- |
+| The New Creator Economy - DAOs, Community Ownership, and Cryptoeconomics | <a href="https://dev.to/dabit3/the-new-creator-economy-daos-community-ownership-and-cryptoeconomics-lnl">View</a> |
+| The Complete Guide to Full Stack Ethereum Development | <a href="https://dev.to/dabit3/the-complete-guide-to-full-stack-ethereum-development-3j13">View</a>
+| What It Takes To Be A Blockchain Developer | <a href="https://www.techfunnel.com/information-technology/how-to-become-blockchain-developer/">View</a> |
+
+## Other
```
Comment (shuklaritvik06) [inline learning-guide/beginner.md:28]: | Title | Content | Length | 
| --- | :-- | --- |  

Should also be there

**Review by shuklaritvik06 — CHANGES_REQUESTED** [review]

File: learning-guide/beginner.md (hunk 1 above)
Comment (shuklaritvik06) [reply learning-guide/beginner.md:28]: In other part of the md file

Comment (vinzvinci) [reply learning-guide/beginner.md:28]: That's not a video, it is a blog so no need to put length.

Comment (vinzvinci) [reply learning-guide/beginner.md:28]: oh okay, the other category, I thought the blog ^^

**Review by shuklaritvik06 — APPROVED** [review]
LGTM :)

**Comment by vinzvinci** [comment]
Thank you for reviewing @shuklaritvik06 🎉 Will merge now

**Pull request merged**